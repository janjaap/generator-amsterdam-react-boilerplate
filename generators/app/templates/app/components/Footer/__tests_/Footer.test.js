import React from 'react';
import { render, cleanup } from 'react-testing-library';
import Footer from '..';

describe('Footer', () => {
  afterEach(cleanup);

  it('matches the snapshot', () => {
    const { container } = render(<Footer />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the general phone number', () => {
    const { getByText } = render(<Footer />);

    const phoneNumberElement = getByText('14 020');
    expect(phoneNumberElement).not.toBeUndefined();
    expect(phoneNumberElement).toBeInstanceOf(HTMLAnchorElement);
  });
});
