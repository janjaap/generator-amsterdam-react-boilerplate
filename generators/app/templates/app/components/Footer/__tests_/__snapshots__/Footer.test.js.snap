// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `
<div
  class="footer-wrapper no-print"
  data-testid="site-footer"
>
  <div
    class="row bg-darkgrey footer"
  >
    <footer
      class="container Footer__FooterComponent-cs8red-0 btfHny"
    >
      <div
        class="row"
      >
        <div
          class="col-12"
        >
          <h3
            class="c-footer__heading"
          >
            Contact
          </h3>
          <p
            class="u-mb--1"
          >
            Heeft u een vraag over de informatie die u aantreft op Mijn Amsterdam?
            <br />
            Neem dan contact op met de gemeente.
          </p>
          <p
            class="u-mb--1"
          >
            Gemeentelijk informatienummer:
             
            <a
              class="c-link c-link--inherit"
              href="tel:14020"
            >
              14 020
            </a>
            <br />
            op werkdagen van 08:00 tot 18:00 uur.
          </p>
        </div>
      </div>
    </footer>
  </div>
  <div
    class="container grid-below-footer"
  >
    <div
      class="row"
    >
      <div
        class="col-12"
      >
        <nav>
          <ul
            class="links horizontal left"
          >
            <li>
              <a
                href="https://www.amsterdam.nl/privacy/"
              >
                <span
                  class="linklabel"
                >
                  Privacy
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;
